### PR TITLE
Add sendBatchTransactions

### PR DESCRIPTION
**Pull request by matthewjablack**
### Description

Add extra outputs to send transaction ledger provider

This enables you to specify more than one output for a transaction in Bitcoin. 

For example: ` sendBatchTransaction ([{ to: address1, value: value1, data: script1 }, { to: address2, value: value2, data: script2 }])`

### Submission Checklist :pencil:

- [x] Implement sendBatchTransaction

### CODE REVIEW

File: packages/client/lib/Chain.js [hunk 1]
```diff
@@ -197,8 +197,17 @@ export default class Chain {
    * @param {!string} from - The address from which the message is signed.
    * @return {Promise<string>} Resolves with a signed transaction.
    */
-  async sendTransaction (to, value, data, from) {
-    return this.client.getMethod('sendTransaction')(to, value, data, from)
+  async sendTransaction (to, value, data, from, extraOutputs) {
```
Comment (monokh) [inline packages/client/lib/Chain.js]: `extraOutputs` parameter should not be required anymore

File: packages/client/lib/Chain.js [hunk 2]
```diff
@@ -197,8 +197,17 @@ export default class Chain {
...
+    return this.client.getMethod('sendTransaction')(to, value, data, from, extraOutputs)
+  }
+
+  /**
+   * Create, sign & broad a transaction with multiple outputs.
+   * @param {string[]} outputs - to, value, data, from.
+   * @return {Promise<string>} Resolves with a signed transaction.
+   */
+  async sendBatchTransaction (outputs) {
```
Comment (monokh) [inline packages/client/lib/Chain.js]: `outputs` is very bitcoin specific term. `transactions` would be more generic.

**Review by monokh — CHANGES_REQUESTED** [review]

File: packages/client/lib/Chain.js (hunk 1 above)
Comment (matthewjablack) [reply packages/client/lib/Chain.js]: good catch, removed 👍

File: packages/client/lib/Chain.js (hunk 2 above)
Comment (matthewjablack) [reply packages/client/lib/Chain.js]: Good point. Changed `outputs` to `transactions`

**Pull request merged**